src/components/navs/AppNav/AppNavSettings.vue

```diff
@@ -219,9 +235,12 @@ export default defineComponent({
     const appLocale = computed(() => store.state.app.locale);
     const appDarkMode = computed(() => store.state.app.darkMode);
     const appTradeInterface = computed(() => store.state.app.tradeInterface);
-    const connectorName = computed(() => getConnectorName(connector.value?.id));
-
-    const connectorLogo = computed(() => getConnectorLogo(connector.value?.id));
+    const connectorName = computed(() =>
+      getConnectorName(connector.value?.id, provider.value)
+    );
+    const connectorLogo = computed(() =>
+      getConnectorLogo(connector.value?.id, provider.value)
+    );
```

Comment:
This might cause bugs, provider can possibly be udnefined

Reply:
Fixed. Made them return default values if provider is undefined.